content/blog/2024-08-08-bootc-almalinux-heliumos.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Bringing bootc to AlmaLinux"
+type: blog
+author: 
+ name: "Isaac Beverly"
+ bio: "Developer, HeliumOS"
+ image: /users/imbev.jpg
+date: 2024-08-08
+images:
+  - /blog-images/2024/2024-08-08-bootc-almalinux.png
+post: 
+    title: "Bringing bootc to AlmaLinux"
+    image: /blog-images/2024/2024-08-08-bootc-almalinux.png
+---
+
+bootc is a new deployment method that powers [Image mode for RHEL](https://www.redhat.com/en/about/press-releases/red-hat-reimagines-enterprise-linux-ai-future-image-mode-red-hat-enterprise-linux) and [HeliumOS](https://www.heliumos.org/), a new Linux distribution. We are excited to announce that AlmaLinux now offers [bootc images for Intel/AMD(x86_64)](https://quay.io/repository/almalinuxorg/almalinux-bootc?tab=tags).
```

Comment:
What is the relevance of HeliumOS here besides it uses bootc? This doesn't seem appropriate here.

Reply:
bootc support in AlmaLinux was largely motivated by HeliumOS, and there are very few projects currently using bootc.

More context in https://chat.almalinux.org/almalinux/channels/sigvirtcontainer

Reply:
If necessary, this post can be published on the HeliumOS blog instead and the AlmaLinux version of the post rewritten to be more AlmaLinux-focused.

Reply:
I've followed your bootc work in MM/GH, thank you so much for getting that moving for us! 

Could you make it more clear that AlmaLinux is now upstream to your project/HeliumOS? It'd make it a bit more clear how it's tied into bootc/Alma.

Reply:
Sure! I'll also add a starting point for other projects to use the AlmaLinux bootc images.

Reply:
@codyro Is there anything else that needs to be changed?